PlexPy.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#!/usr/bin/env python
```

Comment:
Not at all, its just my editor inserting it, it will be cleaned up before i send a pr.